tests/phpt/error_handler_captures_rethrown_exception_once.phpt

```diff
@@ -1,58 +0,0 @@
---TEST--
-Test that a thrown exception is captured only once if rethrown from a previous exception handler
```

Comment:
Can we restore this test so we capture the current (even if not perfect) behavior, so we stop having another debate like this in the future? :smile:

Reply:
This test has been moved to `error_handler_captures_exception_rethrown_from_previous_handler.phpt`, not sure why it got marked as deleted instead of renamed